aspnetcore/security/key-vault-configuration.md

```diff
@@ -210,25 +210,24 @@ config.AddAzureKeyVault(
 
 | Property         | Description |
 | ---------------- | ----------- |
-| `Client`         | <xref:Microsoft.Azure.KeyVault.KeyVaultClient> to use for retrieving values. |
-| `Manager`        | <xref:Microsoft.Extensions.Configuration.AzureKeyVault.IKeyVaultSecretManager> instance used to control secret loading. |
+| `Client`         | <xref:Azure.Security.KeyVault.Secrets.SecretClient> to use for retrieving values. |
+| `Manager`        | <xref:Azure.Extensions.Aspnetcore.Configuration.Secrets.KeyVaultSecretManager> instance used to control secret loading. |
```

Comment:
The namespace is wrong here and elsewhere it's `Azure.Extensions.AspNetCore.Configuration.Secrets`